pandaserver/dataservice/AdderGen.py

```diff
@@ -620,6 +620,7 @@ def parseXML(self):
                 jsonDict = json.loads(self.data)
                 for lfn in jsonDict:
                     fileData = jsonDict[lfn]
+                    self.logger.debug(f"debug: {str(fileData)}")
```

Comment:
Be careful - in terms of size of the logs - with printing this

Reply:
Thanks, will change this after the WFMS meeting